src/components/failedTaskBanner/index.tsx

```diff
@@ -63,26 +53,26 @@ const countFailedProcesses = (processes: ProcessWithStatus[]): CountFailedProces
 
 export default function FailedTaskBanner() {
     const { runningProcesses, completedProcessIds, useFallback } = useContext(RunningProcessesContext);
-    const [data, , fetchData] = useFailedTaskFetcher<FailedProcess>("processes/");
-    const [failedProcesses, setFailedProcesses] = useState<FailedProcess[]>([]);
-    const [failedTasks, setFailedTasks] = useState(countFailedProcesses([]));
+    const { apiClient } = useContext(ApplicationContext);
+    const [summary, setSummary] = useState<ProcessStatusCounts>();
+    const [failedTasks, setFailedTasks] = useState<CountFailedTasks>(countFailedTasks(summary));
     const history = useHistory();
     const handleOnClick = useCallback(() => history.push("/tasks"), [history]);
 
-    useHttpIntervalFallback(useFallback, () => fetchData(0, 10, [], filterFailedTasks));
+    useHttpIntervalFallback(useFallback, () => getSummary());
 
     useEffect(() => {
-        const newList = [...data, ...runningProcesses.filter((p) => failedProcessStatuses.includes(p.last_status))];
-        setFailedProcesses(newList.filter((p) => !completedProcessIds.includes(p.pid)));
-    }, [data, runningProcesses, completedProcessIds]);
+        setFailedTasks(countFailedTasks(summary));
+    }, [summary]);
 
-    useEffect(() => {
-        setFailedTasks(countFailedProcesses(failedProcesses));
-    }, [failedProcesses]);
+    const getSummary = () =>
+        apiClient.processStatusCounts().then((res) => {
+            setSummary(res);
+        });
 
     useEffect(() => {
-        fetchData(0, 10, [], filterFailedTasks);
-    }, []); // eslint-disable-line react-hooks/exhaustive-deps
+        getSummary();
+    }, [runningProcesses, completedProcessIds]);
```

Comment:
Build fails on this as it expects `getSummary` as a dependency, but when I add this it goes into an infinite loop. Not sure yet how to fix this (old code resorted to an `eslint-disable-line` as well)